.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+  - "stable"
+branches:
+  only:
+    - master
```

Comment:
What does this do?

Reply:
@abayo-luc
It specifies the branch to build, or you can specify the commit to build. [more info](https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches)

I have removed I found that it is not adding any value to our build system.
Thanks